src/lib/internationalization/locales/en.cts

```diff
@@ -388,7 +388,8 @@ export = {
         "hostedBaseUrl must start with http:// or https://",
     useHostedBaseUrlForAbsoluteLinks_requires_hostedBaseUrl:
         "The useHostedBaseUrlForAbsoluteLinks option requires that hostedBaseUrl be set",
-    favicon_must_be_ico_or_svg: "Favicon file must be either a .ico or .svg",
+    favicon_must_be_ico_or_svg:
```

Comment:
This is getting long enough that I think it should be made a template: `favicon_must_have_one_of_the_following_extensions_0`

Reply:
done